ChangeLog.md

```diff
@@ -1,3 +1,7 @@
+## 0.1.0.2
```

Comment:
I'll fix the version bump, it should be 0.1.1 since this introduces a new API in a non-breaking way.